### PR TITLE
gnrc_slip: set NETOPT_IS_WIRED

### LINKED ISSUE #4168: gnrc_slip: should set NETOPT_IS_WIRED
`ifconfig` currently lists the slip interface as "wireless" which is, for the most common set ups, wrong.

```
2015-10-27 14:35:05,418 - INFO # > ifconfig
...
2015-10-27 14:35:05,464 - INFO # Iface  6  
2015-10-27 14:35:05,466 - INFO #            MTU:1280  
2015-10-27 14:35:05,469 - INFO #            Link type: wireless
2015-10-27 14:35:05,474 - INFO #            inet6 addr: ff02::1/128  scope: local [multicast]
2015-10-27 14:35:05,475 - INFO #            

```

### PR DESCRIPTION
**Pull request by authmillenon**
Fixes #4168

### CODE REVIEW

File: sys/net/gnrc/link_layer/slip/gnrc_slip.c [hunk 1]
```diff
@@ -185,6 +185,11 @@ static void _slip_send(gnrc_slip_dev_t *dev, gnrc_pktsnip_t *pkt)
     gnrc_pktbuf_release(pkt);
 }
 
+static inline int _slip_get(gnrc_netapi_opt_t *opt)
+{
+    return (opt->opt == NETOPT_IS_WIRED) ? 1 : -ENOTSUP;
```
Comment (gebart) [inline sys/net/gnrc/link_layer/slip/gnrc_slip.c]: please expand to a multiline `if`, it makes it clearer to read and easier to expand this further in the future if we want to add another netopt to slip.

Comment (authmillenon) [inline sys/net/gnrc/link_layer/slip/gnrc_slip.c]: Done

**Comment by gebart** [comment]
ifconfig still says the slip interface is wireless afaics... @authmillenon do you know what is missing?

**Comment by authmillenon** [comment]
Don't know, but am investigating.

**Comment by authmillenon** [comment]
It's an unrelated error: see #4177.

**Comment by gebart** [comment]
ACK, please squash
Tested together with #4177 on mulle, works fine.

**Comment by OlegHahm** [comment]
So you have a mulle-based border router?

**Comment by gebart** [comment]
only tested the slip part with ping6 from linux, but yeah, I used the examples/gnrc_border_router example and a FTDI USB-UART cable connected on UART_1 with tunslip6 on the linux side. I will do some more tests with RPL etc. when I have some spare time.

**Comment by authmillenon** [comment]
Squashed

**Comment by OlegHahm** [comment]
cool

**Pull request merged**